contracts/VAIController.sol

```diff
@@ -95,6 +101,12 @@ contract VAIController is VAIControllerStorageG2, VAIControllerErrorReporter, Ex
 
             address minter = msg.sender;
 
+            (vars.mathErr, totalMintedVAI) = addUInt(totalMintedVAI, mintVAIAmount);
+
+            if (mintCap != 0) {
```

Comment:
do we need to handle a case where the mintCap is 0? We could throw the same error in this case too

Reply:
@coreyar In vToken borrow caps we are using 0 for infinite borrow. But as discussed in DSU, I will changed it to such that 0 mean no minting is allowed and by default infinite minting is enabled.

Reply:
@kkirka I agree. I have updated the logic.